src/contract.js

```diff
@@ -47,12 +51,31 @@ class Contract {
   getSourceCode(hash, filterOptions) {
     if (is.notHash(hash)) return Promise.reject(new Error(NO_ADDRESS))
     return get(this.web3data, {
-      hash,
+      pathParam: hash,
       endpoint: ENDPOINT,
       subendpoint: 'source-code',
       filterOptions
     })
   }
+
+  async getCode(hash) {
+    const response = await this.getDetails(hash)
+    return new Promise((resolve, reject) => {
+      if (
+        is.null(response) ||
+        is.undefined(response) ||
+        response.status !== 200
+      ) {
+        reject(new Error('/contracts/:hash failed to respond'))
```

Comment:
@taylorjdawson this is revealing the API endpoints rather than just reporting the issue. Let's use:
`Failed to retrieve contract`